code/datums/ai/objects/item_behaviors.dm

```diff
@@ -1,13 +0,0 @@
-
-///This behavior is for obj/items, it is used to free themselves out of the hands of whoever is holding them
-/datum/ai_behavior/item_escape_grasp
-
-/datum/ai_behavior/item_escape_grasp/perform(delta_time, datum/ai_controller/controller)
```

Comment:
These should be migrated to an unused dm and checked, these are useful.

Reply:
Useful for what, this is just a copy of something that exists in another file

Reply:
wait why wasn't this erroring on duplicate defines already

Reply:
because this file was unincluded in the tgstation.dme

Reply:
What MDB said 😄